_posts/2024-01-19-yannick-horrix-opentofu.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+authors: [yannick-horrix]
+title: 'Choosing Your Orchestrator: OpenTofu vs Terraform'
+image: /img/2024-01-19-yannick-horrix-opentofu/banner.jpg
+tags: [ cloud, automation, terraform, opentofu, iac ]
+category: cloud
+comments: true
+---
+
+> The power of Open Source is the power of the people. The people rule.
+
+
+{:.no_toc}
+- TOC
+{:toc}
+
+# Introduction
+Terraform, an industry-leading infrastructure as code (IaC) tool, has been a cornerstone of cloud provisioning and infrastructure management since its inception in 2014. 
+Over the years, it has fostered a vibrant community comprising thousands of users, contributors, vendors, and an extensive ecosystem of open source modules and extensions.
+
+However, August 10, 2023, marked a significant shift in Terraform’s trajectory when HashiCorp, its stewarding organization, changed the license from the Mozilla Public License (v2.0) to the Business Source License (v1.1). 
+This abrupt change sent ripples across the Terraform community, sparking concerns about its future.
```

Comment:
Discuss impact of the license change, especially w.r.t. the uncertain future of the "open-source"-ness and Free Open Source Software-ness of the product/solution

Reply:
Reference that this will be discussed later (just read it a few paragraphs down)